base_continent/__manifest__.py

```diff
@@ -1,13 +1,17 @@
 # -*- coding: utf-8 -*-
 # © 2014-2016 Camptocamp SA (Author: Romain Deheele)
+# © 2017 senseFly, Amaris (Author: Quentin Theuret)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 
 {
     'name': 'Continent management',
-    'version': '8.0.1.0.1',
-    'depends': ['base'],
-    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'version': '10.0.1.0.1',
+    'depends': [
+        'base',
```

Comment:
No need for this if you depend on other thing

Reply:
otoh, why do you depend on sales team now?

Reply:
Because some menus (Localisation)  are moved to sales_team menu.

Reply:
We (with @alexis-via) had the same discussion, and decided to move into technical settings. So you don't force to install an undesired module.
ref : https://github.com/OCA/l10n-france/pull/131#issuecomment-364285722